ROADMAP.md

```diff
@@ -51,24 +79,25 @@ hole punching stack.
 ## Improved Wasm support
 
 | Category             | Status | Target Completion | Tracking                                          | Dependencies | Dependents                                   |
-|----------------------|--------|-------------------|---------------------------------------------------|--------------|----------------------------------------------|
-| Developer ergonomics | todo   | Q3/2023           | https://github.com/libp2p/rust-libp2p/issues/2617 |              | [WebRTC](#webrtc-support-browser-to-browser) |
+|----------------------+--------+-------------------+---------------------------------------------------+--------------+----------------------------------------------|
+| Developer ergonomics | todo   |                   | https://github.com/libp2p/rust-libp2p/issues/2617 |              | [WebRTC](#webrtc-support-browser-to-browser) |
 
 The project supports Wasm already today, though the developer experience is cumbersome at best.
 Properly supporting Wasm opens rust-libp2p to a whole new set of use-cases. I would love for this to
 happen earlier. Though (a) I think we should prioritize improving existing functionality over new
 functionality and (b) we don't have high demand for this feature from the community. (One could
 argue that that demand follows this roadmap item and not the other way round.)
 
-## WebRTC support (browser-to-browser)
+## WebRTC in the browser via WASM
 
 | Category     | Status | Target Completion | Tracking                                   | Dependencies                                                                              | Dependents |
-|--------------|--------|-------------------|--------------------------------------------|-------------------------------------------------------------------------------------------|------------|
-| Connectivity | todo   | Q3/2023           | https://github.com/libp2p/specs/issues/475 | [Improved WASM support](#improved-wasm-support), https://github.com/libp2p/specs/pull/497 |            |
-
+|--------------+--------+-------------------+--------------------------------------------+-------------------------------------------------------------------------------------------+------------|
+| Connectivity | todo   |                   | https://github.com/libp2p/specs/issues/475 | [Improved WASM support](#improved-wasm-support), https://github.com/libp2p/specs/pull/497 |            |
 
-Once WebRTC for browser-to-server is complete, we can begin work on **browser-to-browser** and complete the WebRTC connectivity story.
-We need to improve rust-libp2p's WASM story first.
+Use the browser's WebRTC stack to support
+[`/webrtc`](https://github.com/libp2p/specs/blob/master/webrtc/webrtc.md) and
+[`/webrtc-direct`](https://github.com/libp2p/specs/blob/master/webrtc/webrtc-direct.md) from within
+the browser using rust-libp2p compiled to WASM.
```

Comment:
I'd be great to add a few more words here on why this is cool. It will allow people to use `rust-libp2p` for their entire system, i.e. clients in the browser and servers!

Reply:
> I'd be great to add a few more words here on why this is cool. It will allow people to use `rust-libp2p` for their entire system, i.e. clients in the browser and servers!

Sounds familiar. PeerTube?

Reply:
Good idea. Added with latest commit.